src/SlateAsInputEditor/index.js

```diff
@@ -63,15 +62,9 @@ const ToolbarWrapper = styled.div`
   box-shadow: ${props => props.TOOLBAR_SHADOW || 'none'};
 `;
 
-const Heading = ({ type, children, attributes }) => createElement(
-  styled(type)`
-      font-family: 'serif';
-    `,
-  {
-    'data-key': attributes['data-key']
-  },
-  children,
-);
+const Heading = styled.div`
+  font-family: serif;
+`
```

Comment:
I believe we needed this for the `data-key` in order to navigate through `cicero-ui`'s `Navigation` component

Reply:
I'm spreading the attributes like this
```
return <Heading as="h1" {...attributes}>{children}</Heading>;
```
Shouldn't this be enough as `attributes` object already contains the `data-key`?

Reply:
I can revert back to the `createElement` here also. I implemented this there was a focusing issue. But now the focus management is now explicit, via DOM and slate apis so I hope it will work with the previous version too.
I left it as is because it is shorter and simpler.

Reply:
I tried this locally and it still works... This comes from #92 and #136, I must just have gotten lost on why these changes needed to be made in the first place.

Reply:
Yes. #92 does pass any prop into createElement at all. And #136 fixes this by adding it explicitly.

So should I leave this as it is, right?

Reply:
I believe so, yes.